### ISSUE TITLE
Can't compile the code because of Chinese letters

### ISSUE THREAD

**Issue by ghost** [Microsoft/ILMerge#1]
I wonder why there are a lot of Chinese letters in the code. A try to obfuscate or what?
Because of these Chinese letters I get a lot of errors and can't compile.

Image: http://i.imgur.com/BwziSFx.png

**Comment by mike-barnett** [comment]
There shouldn't be any chinese characters in the code. I don't know how you could be seeing any. Can you provide more details?

**Comment by naitp** [comment]
I am also having this issue.  I downloaded the master branch as a zip file, decompressed, and built the application, and received 75 build errors all in System.Compiler/GlobalSuppressions.cs (https://github.com/Microsoft/ILMerge/blob/master/System.Compiler/GlobalSuppressions.cs). I can copy and paste the contents of the file into my version and get a successful build.

Just to be sure, I cloned a personal fork from github to my local machine and got the same error.

If I attempt to edit the file in my fork in Chrome, I get the message: 
"We've detected the file encoding as UTF-16LE. When you commit changes we will transcode it to UTF-8."  The first character in the file is \ufeff (ZERO WIDTH NO-BREAK SPACE).  Deleting this leading character, committing via the browser, and pulling the code to my local system fixes the issue.  I will create a pull request for this fix.
